src/transport/scheduler/xqc_scheduler_minrtt.c

```diff
@@ -103,24 +98,20 @@ xqc_minrtt_scheduler_get_path(void *scheduler,
                 best_path[path_class] ? best_path[path_class]->path_id : -1);
     }
 
-
-    for (path_class = XQC_PATH_CLASS_AVAILABLE_HIGH; 
-         path_class < XQC_PATH_CLASS_PERF_CLASS_SIZE; 
-         path_class++)
+    if (best_path_class != XQC_PATH_CLASS_PERF_CLASS_INVALID
```

Comment:
The original code intends to find the path with the minimal RTT for each path class and selects the one with the lowest path class index. You modification breaks that.

Reply:
ok, but I don’t understand why you need to choose the first path you come across with a minimum RTT value with a lowest path class index, if there is a possibility that further there will be a path with an even lower RTT, albeit with a bigger path class index

Reply:
thus, as long as this probability exists, `xqc_scheduler_minrtt.c` will return the path not with the smallest RTT